guide/app_accuracy.tex

```diff
@@ -33,9 +33,9 @@ \chapter{Accuracy}
 the Solar gravity field, may still be required for critical
 observations of stellar occultations by asteroids, grazing
 occultations of stars by the Moon or similar things. Stellarium does
-not compute these effects, and is definitely not capable for detailed
```

Comment:
OK, if we are changing something, admittedly the program is not capable, humans may be. So "not usable for..."